### ISSUE TITLE
OpsWorks stays on "configure" step forever

### ISSUE THREAD

**Issue by svasquezm** [aws/opsworks-cookbooks#84]
Lately I use OpsWorks to run instances with my customs cookbooks and recipes. In particular, one of them runs a Instance with a **remote_file** Chef command. This command download a big file (~ 350MB) from a web URL. I dont know but that recipe is the only one staying on **configure** step.

The **/var/lib/aws/opsworks/chef/XXXX-XX-XX-XX-XX-XX-XX.log** log file says:
[2013-12-02T17:37:32+00:00] INFO: Chef Run complete in 152.972258 seconds
[2013-12-02T17:37:32+00:00] INFO: Running report handlers
[2013-12-02T17:37:32+00:00] INFO: Report handlers complete
[2013-12-02T17:37:32+00:00] DEBUG: Exiting

That means recipe finished successfully.

And the **/var/log/aws/opsworks/opsworks-agent.process_command.log** file says
[2013-12-02 18:12:04] ERROR [opsworks-agent(850)]: process_command: Error during perform: OpenSSL::SSL::SSLError - SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed - /usr/lib/ruby/1.8/net/http.rb:586:in `connect'
/usr/lib/ruby/1.8/net/http.rb:586:in`connect'
/usr/lib/ruby/1.8/net/http.rb:553:in `do_start'
/usr/lib/ruby/1.8/net/http.rb:548:in`start'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/net/http/connection_pool/session.rb:118:in `start'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/net/http/connection_pool.rb:208:in`_create_session'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/net/http/connection_pool.rb:193:in `session_for'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/net/http/connection_pool.rb:171:in`request'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/net/http/connection_pool/connection.rb:173:in `request'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/http/net_http_handler.rb:66:in`handle'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/client.rb:233:in `make_sync_request'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/client.rb:259:in`retry_server_errors'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/client.rb:226:in `make_sync_request'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/client.rb:451:in`client_request'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/client.rb:334:in `log_client_request'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/client.rb:420:in`client_request'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/client.rb:316:in `return_or_raise'
/opt/aws/opsworks/current/vendor/bundle/ruby/1.8/gems/aws-sdk-1.8.1.2/lib/aws/core/client.rb:419:in`client_request'

In summary, the recipe finished and install everything, but Opsworks still on **configure** step and the consequence is I cannot execute more commands via **run_command**

Please, help.

**Comment by awsrequena** [comment]
Is this problem still relevant?

(close this after 01.09.2014)

**Comment by DanielRedOak** [comment]
Seeing a very similar error, although in setup state.  Have not found the root cause, have you @svasquezm ?

**Comment by DanielRedOak** [comment]
state=SSLv3 : Is this SSLv3 related?

**Comment by jweiss** [comment]
closing as this is a very old issue
